frontend/ui/widget/opdsbrowser.lua

```diff
@@ -20,6 +20,7 @@ local http = require('socket.http')
 local https = require('ssl.https')
 local ltn12 = require('ltn12')
 local mime = require('mime')
+local util = require("util")
```

Comment:
Oh, and if you happen to have the time there's a new part of the coding style about the order of requires: https://github.com/koreader/koreader-base/wiki/Coding-style#order-of-requires

But only if you feel like it. ;-)